content/configuration/tasks-deploy.md

```diff
@@ -185,6 +185,21 @@ specification:
             memory: 500Mi
 ```
 
+## Replicas and Redundancy 
+
+{{< note title="Note" >}}
+For background, a pod comprises a Kubernetes service and container(s). A replica is copy of a pod running on a different node, where a node is a worker machine within a cluster.
+{{< /note >}}
+
+By default a pod is configured to have a single replica. You may want to provide redundancy for a pod across multiple nodes to make your application more resilient. This can be done by increasing the number of replicas:
```

Comment:
> a pod is configured to have a single replica

The "pod" per say is, in Kubernetes terminology, a single running instance. We could use the term "deployment" to refer to the abstraction of the multiple replicas. "services" is used in CP's and Docker-Compose YAML files.